crispy_bulma/templates/bulma/widgets/file_upload_input.html

```diff
@@ -1,17 +1,33 @@
+{% if widget.is_initial %}
+  {{ widget.initial_text }}:
+  <a href="{{ widget.value.url }}">
+    {{ widget.value }}
+  </a>
+  {% if not widget.required %}
+  <input type="checkbox" name="{{ widget.checkbox_name }}" id="{{ widget.checkbox_id }}"{% if widget.attrs.disabled %} disabled{% endif %}>
+  <label for="{{ widget.checkbox_id }}">
+    {{ widget.clear_checkbox_label }}
+  </label>
+  {% endif %}
+{% endif %}
 <div class="field">
-  <div class="file is-primary has-name">
+  <div class="file is-primary has-name is-small">
     <label class="file-label">
-      <input type="file" name="{{ widget.checkbox_name }}"{% include "django/forms/widgets/attrs.html" %}>
+      <input type="file" name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %} onchange="document.getElementById('{{ widget.attrs.id }}_label').textContent=this.value;">
```

Comment:
Bulma does not include javascript and this template pack won't either, it's out of scope of this project, so the onchange event needs to go.

Thanks for the fixes, I'm still testing it and if everything else is OK I'll merge it and make a new release.

Reply:
I make some update
I removed the JS code and proposed a way to add it easily (optional)

Reply:
@pythonbrad thanks for the PR! I've merged it with small modifications.